lambda/lambda_py/notifiers/sendgrid_notifier.py

```diff
@@ -15,11 +15,17 @@ class SendGridNotifier(IWebhookNotifier):
     - DEST_EMAIL_ADDRESSES: ',' separated lists of recipient email addresses
     """
 
-    def __init__(self):
-        super(SendGridNotifier, self).__init__()
-        self.api_key = self.param("API_KEY")
-        self.source_email_address = self.param("SOURCE_EMAIL_ADDRESS")
-        self.dest_email_addresses = self.param("DEST_EMAIL_ADDRESSES").split(",")
+    def __init__(
+        self,
+        api_key: str,
+        source_email_address: str,
+        dest_email_addresses: List[str],
+        **_kwargs,
```

Comment:
I never encountered this syntax of _kwargs ?

Reply:
The leading `_` is just to indicate that the argument is not used.

Reply:
@Lotram what is your opinion on that ?

Reply:
I've never seen it either. IMO `**kwargs` is more usual, even if it's unused

Reply:
Ok reverted to `kwargs`